src/features/MentorPage/MentorsFilter/Skills/PageButton.tsx

```diff
@@ -0,0 +1,52 @@
+import styled, { css } from 'styled-components';
+import * as cssVariables from '../../../../components/variables';
+import { DOTS } from './usePagination';
```

Comment:
>  Maybe it could at least be in some kind of constants file instead of another feature/component file.

I'll argue against this. I think it is better to export the constant from the relevant module. For example the DOTS-variable is first introduced in the usePagination-hook. 

I would like to have the code divided by features, not by types. I would rather not have a "hooks"-folder for hooks, "constants"- for constants and "reducers" - for reducers. Instead gather the relevant code together as closely in the file-tree as possible. It's easier to refactor, find code and so on. 

This philosophy means that whenever you need a type, write it in the place where it needs to be introduces. You need that type somewhere else? Then export it. When you need it in even more places, then it is time to move it to a common folder. But this variable is tied to the usePagination-hook, so I'd have it there.
If we need to have pagination in multiple places, then we can move the hook to a common place, and then the DOTS-variable will migrate at the same time.

Calling it THREE_DOTS is fine by me. I guess it makes more sense =)